src/common/signalk/SKNMEAVisitor.cpp

```diff
@@ -27,19 +27,85 @@
   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
   THE SOFTWARE.
 */
-
+#include <Arduino.h>
```

Comment:
Why do you need `Arduino.h` here? It's a very big header that includes many other headers and it makes it hard to compile code on other platforms. 

I try to always be more specific about what I need and only include the specific header that I need.